src/main/js/helpers/axis.js

```diff
@@ -191,37 +191,136 @@ const getAxesScale = (axis, scale, config) => {
         getAxisInfoOrientation(config.axis.x.orientation)
     );
 
-    // If ticksCount is undefined or greater than TICKS_MAXCOUNT
-    // AND if the Y2 is visible, then utilize a default value for
-    // the ticksCount. This is based on the ranges of the Y & Y2 axes.
+    if (hasY2Axis(config.axis)) {
+        if (!utils.isUndefined(config.axis.y.ticks.values)) {
```

Comment:
We can check for y2 ticks, but since we are already checking for the y2 axis, it does not change the behavior very much. Not checking for y2 also allows for the additional situation where a consumer only provides Y ticks and y2 ticks are automatically generated. This way, they can have a starting point for y2 ticks and make any changes necessary to align them with the Y ticks.